_episodes_rmd/02-r-plotting.Rmd

```diff
@@ -568,38 +606,38 @@ To start, we will read in the data:
 > What argument should be provided in the below code to read in the full dataset?
 >
 > ```{r ImportFullData, eval = FALSE}
-> gapminder_data <- read_csv()
+> smoking <- read_csv()
 > ```
 >
 > > ## Solution
 > >
 > > ```{r ImportFullDataSolutionEcho, eval = FALSE}
-> > gapminder_data <- read_csv("data/gapminder_data.csv")
+> > smoking <- read_csv("data/smoking_cancer.csv")
 > > ```
 > {: .solution}
 {: .challenge}
 
 Let's take a look at the full dataset. We could use `View()`, the way we did for the smaller dataset, but if your data is too big, it might take too long to load. So let's just look at it in the console. Note that if you're working with something other than a tibble, you might have to use the `head()` function to take a look at the first few row, but when you print out a tibble it will just print out the first few lines:
 
 ```{r InitialExploration, eval = FALSE}
```

Comment:
Hmm because we have this to `eval = FALSE` we don't see the first few lines on the website. Maybe change to `eval = TRUE`?

OR maybe we should actually move the description above about head etc. to the 1st dataset we read in, since that's where we talk about printing it out to the console. And then we don't have to re-explain View again here, we can just look at it in the console like we explained before.

Reply:
I kind of like it the way we have it now, so that we don't overwhelm everyone at the top. So I changed eval = TRUE.